pkgs/applications/networking/instant-messengers/telegram/tdesktop/default.nix

```diff
@@ -70,7 +74,6 @@ in mkDerivation rec {
     # We're allowed to used the API ID of the Snap package:
     "-DTDESKTOP_API_ID=611335"
     "-DTDESKTOP_API_HASH=d524b414d21f4d37f08684c1df41ac9c"
-    #"-DDESKTOP_APP_SPECIAL_TARGET=\"\"" # TODO: Error when set to "": Bad special target '""'
     "-DTDESKTOP_LAUNCHER_BASENAME=telegramdesktop" # Note: This is the default
```

Comment:
Maybe drop that line as well? It's up to upstream how to name the desktop file, isn't it? I think the main reason this option exist are flatpak and snap packages.

Reply:
Removed. Seems `telegramdesktop` is just the default value.